pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -191,26 +188,19 @@ def check_checksum(archive: Path, source_metadata: dict[str, Any]) -> None:
     source_metadata
         The source section from meta.yaml.
     """
-    checksum_keys = {"md5", "sha256"}.intersection(source_metadata)
```

Comment:
I dropped support for md5, since as far as I can tell we are no longer using it.